src/test/java/org/jabref/logic/integrity/PersonNamesCheckerTest.java

```diff
@@ -62,4 +69,29 @@ public void validCorporateNameAndPerson() throws Exception {
         assertEquals(Optional.empty(), checker.checkValue("{JabRef} and Stefan Kolb"));
         assertEquals(Optional.empty(), checker.checkValue("{JabRef} and Kolb, Stefan"));
     }
+
+    @ParameterizedTest
+    @MethodSource("provideCorrectFormats")
```

Comment:
This is a good step forward. 

The aspect of a `field` check is missing. To keep that in `IntegrityCheckTest` is a good idea. See my comments there.

The inputs will be duplicated, but that is OK.